contract-examples/test/tx_allow_list.ts

```diff
@@ -19,13 +20,55 @@ const ROLES = {
   ADMIN: 2
 }
 
+const testFn = (fnName, overrides = {}, debug = false) => async function () {
```

Comment:
Ignore the duplication for now, will find the common abstraction before this is ready to merge